src/core/log.js

```diff
@@ -6,23 +6,37 @@ const {multistream} = require('pino-multi-stream');
 const ecsFormat = require('@elastic/ecs-pino-format');
 const config = require('config');
 const {resolveHome} = require('../core/utils');
-
+const {getNetworkAddress} = require('../wallet/keystore');
 const datadir = config.get('datadir');
 const {level, enabled, sendLogs, sendLogsTo} = config.get('log');
 const options = {enabled, formatters: ecsFormat(), level: pino.levels.values[level]};
 const streams = [];
+const {RUNNING_PKG_MODE} = require('../index');
+const DEFAULT_LOG_LEVE_FOR_PRODUCTION = 'info';
 
 if (sendLogs && sendLogsTo) {
     const [address, port] = sendLogsTo.split('://').pop().split(':');
     streams.push(new udpTransport({address, port}));
 }
 
 streams.push(
-    {level: options.level, stream: pino({prettyPrint: {colorize: true}})[pino.symbols.streamSym]},
+    {
+        level: RUNNING_PKG_MODE ? DEFAULT_LOG_LEVE_FOR_PRODUCTION : options.level,
```

Comment:
We don't need to do this ternary here, as well as hard code the log level using `DEFAULT_LOG_LEVE_FOR_PRODUCTION` variable in this file. We should leverage `config` instead. Just override the config level in [this file](https://github.com/pointnetwork/pointnetwork/blob/develop/config/production.yaml), the same way it is set in the default config [here](https://github.com/pointnetwork/pointnetwork/blob/develop/config/default.yaml#L20). Then the dashboard will simply set the mode to `production` and it will work as intended.

Reply:
so basically we should add a production.yaml file?

Reply:
It is already added, just let's specify the log level there explicitly.